### PR TITLE
✨ add clipboard feature for social share

### LINKED ISSUE #26: Copy social link
Add a bit of JS to implement the “Copy this link” in the `Social Share` component

### PR DESCRIPTION
**Pull request by Yago**
will close #26

### CODE REVIEW

File: assets/components/organisms/social-share/social-share.scss [hunk 1]
```diff
@@ -51,10 +51,28 @@
...
+      box-shadow: none !important;
+      &:before { content: none; }
+    }
+  }
 
   @include media-breakpoint-up(lg) {
     margin-top: 0;
   }
+
+  @include media-breakpoint-down(md) {
```
Comment (knuch) [inline assets/components/organisms/social-share/social-share.scss]: maybe a bit of mobile styling to make it more elegant? 
capture d ecran 2018-03-29 a 11 41 49

Comment (knuch) [reply assets/components/organisms/social-share/social-share.scss]: also the button top-margin seems a bit visually big. Maybe we should diminish it globally? Or maybe it is just that word that makes it look so.

Comment (Yago) [reply assets/components/organisms/social-share/social-share.scss]: Which “top-margin” ? Yes we can to a bit of mobile enhancement, but later. We don't have any mobile design for this component and we are not sure about the amount of social button (G+ is dead...) We will plan a fine tunning round with Guillaume soon 😉

Comment (knuch) [reply assets/components/organisms/social-share/social-share.scss]: I meant button top-padding :D Okay for mobile design

**Pull request merged**